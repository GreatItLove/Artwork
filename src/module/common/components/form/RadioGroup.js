import React from 'react';
import { RadioButtonGroup } from 'material-ui/RadioButton';

/**
 * @function RadioGroup
 * @param props
 * @returns {XML}
 */
function RadioGroup({ input, ...rest }) {
  return (
    <RadioButtonGroup
      {...input}
      {...rest}
      valueSelected={input.value}
      onChange={(event, value) => input.onChange(value)}
    />
  );
}

export default RadioGroup;
