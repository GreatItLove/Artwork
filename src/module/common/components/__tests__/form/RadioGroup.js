import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import RadioGroup from '../../form/RadioGroup';

describe('RadioGroup', () => {
  it('Should be same snapshot', () => {
    const input = {};
    const name = 'somename';
    const wrapper = shallow(<RadioGroup {...{ input, name }} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
