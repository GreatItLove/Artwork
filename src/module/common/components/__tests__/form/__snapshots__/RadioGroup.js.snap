// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioGroup Should be same snapshot 1`] = `
<RadioButtonGroup
  name="somename"
  onChange={[Function]}
  style={Object {}}
/>
`;
